hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/UnsupportedContentTypeException.java

```diff
@@ -22,4 +23,12 @@ public UnsupportedContentTypeException(Subscription subscription) {
         ));
     }
 
+    public UnsupportedContentTypeException(String payloadContentType, Schema schema) {
```

Comment:
why Avro Schema here? i don't get it, we probably want topic name, not schema name